### ISSUE TITLE
Platoon request antispam

### ISSUE THREAD

**Issue by Sunshine40** [PavelMaca/WoT-BattleMessenger#5]
I don't know how it goes with other servers, but on WoT CN (hosted by wot.kongzhong.com), spamming of platoon request is extremely annoying. A mod that can filter platoon requests would be appreciated.

How I'd like the mod to work:
1. Custom words filter to block platoon requests.
2. Block any platoon request with a message longer than x characters.
3. Friends (in the "friend list" of the account) are auto-whitelisted. (This has lower priority since a friend hardly ever needs to send a long message along with a platoon request).

BTW, it would be even better if in-garage chat messages could be filtered. I believe over 95% of spamming on WoT CN would be blocked by using a single keyword "prize" (in Chinese).
(There's still 1 in 1000 people that would believe a pop-up message saying "You've won (some kind of) lottery". Frauds spam everyone repeatedly hunting for those silly/naive guys, making everyone disgusted with their spam messages on WoT CN)

Is this feature possible to get implemented? Could it be done in the near future? Please let me know. Thanks in advance!
